src/NLog/LogFactory.cs

```diff
@@ -821,7 +819,7 @@ internal void ReloadConfigOnTimer(object state)
 
                     if (newConfig is XmlLoggingConfiguration xmlConfig)
                     {
-                        if (!xmlConfig.InitializeSucceeded.HasValue || !xmlConfig.InitializeSucceeded.Value)
+                        if ((xmlConfig.InitializeSucceeded ?? false) == false)
```

Comment:
I prefer

`if(xmlConfig.InitializeSucceeded  != true)`

or 

`if(!(xmlConfig.InitializeSucceeded ?? false))`

(`== true` or `==false` is a common code smell)

Reply:
Changed as suggested

Reply:
Guess this needs to be pushed? It's still shows `== false` ;)